app/assets/javascripts/services/post_service.js

```diff
@@ -6,18 +6,18 @@ ManageIQ.angular.app.service('postService', ["miqService", "$timeout", "$window"
       angular.toJson({
         action: "edit",
         resource: updateObject
-      })).then(handleSuccess, handleFailure);
+      })).then(handleSuccess, miqService.handleFailure)
```

Comment:
Can you change this to simply --
```javascript
.then(handleSuccess)
```
No need for `miqService.handleFailure` in `.then`
(Sorry, I should have specified that before)

And the same change below.